build/commands/lib/config.js

```diff
@@ -476,6 +484,9 @@ Config.prototype.buildArgs = function () {
     // set goma_dir to the redirect cc output dir which then calls gomacc
     // through env.CC_WRAPPER
     args.goma_dir = path.join(this.nativeRedirectCCDir)
+  } else if (this.useRemoteExec) {
```

Comment:
should we list this first so it takes precedence over goma settings?

Reply:
> should we list this first so it takes precedence over goma settings?

`gn gen` will fail if both goma and remoteexec are enabled, so the user will be forced to pick one.

Reply:
Yes, but I'm saying that might not be a particularly desirable failure as even testing this will force people to modify npm config

Reply:
> Yes, but I'm saying that might not be a particularly desirable failure as even testing this will force people to modify npm config.

For testing people can do `npm run build -- --use_goma=false --use_remoteexec`. This will override `brave_use_goma=true` set it `.env`.

>  Maybe for now it should default to goma unless you explicitly force reclient on the command line?

You want this to work like so: `--use_goma --use_remoteexec` will use `remoteexec`, but `--use_goma` will just use `goma`? We can do that, but this may silence an incorrect configuration. It will be unclear what exactly is in use currently.

Reply:
I guess as long as there's some way to override it without changing npm config that's fine